core/kernel.rbs

```diff
@@ -2267,10 +2267,8 @@ module Kernel : BasicObject
   #     enum.first(4) # => [1, 1, 1, 2]
   #     enum.size # => 42
   #
-  def enum_for: (Symbol method, *untyped, **untyped) ?{ (?) -> Integer } -> Enumerator[untyped, untyped]
-              | () ?{ () -> Integer } -> Enumerator[untyped, self]
```

Comment:
this `, self` is actually incorrect; if `method` isn't given it defaults to `each`, and the return value is whatever `each` returns (which is traditionally `self` but not always)